crates/teloxide/src/backoff.rs

```diff
@@ -2,14 +2,30 @@ use std::time::Duration;
 
 pub type BackoffStrategy = Box<dyn Send + Fn(u32) -> Duration>;
 
+const THRESHOLD_ERROR_QUANTITY: u32 = 6;
```

Comment:
what's the point of the const? given that it's documented elsewhere (in the use) and only used once, it seems like here it's nicer to just have the number